iree_special_models/sd3/mmdit/test_mmdit.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers import *
+import os
+
+repo_root = os.getenv("TEST_SUITE_REPO_ROOT")
+current_dir = repo_root + "/iree_special_models/sd3/mmdit"
```

Comment:
This is brittle (if the file is moved, this line will also need to be updated). How about either `os.path.dirname(__file__)` or `Path(__file__).parent`?

Reply:
Yeah that was how it was before, but to be able to have this implemented downstream we will have to go with this for now I think. Because, these python files will be in different places than where we are cloning the SHARK-TestSuite repo right?

Reply:
Consider how much code is forked vs used directly then decide on a setup from there.

This is only used here to get the path to `attention_and_matmul_spec.mlir`. If the entire test file is forked (so any flags and configurations can be changed), then there's no need to try making this something that can be substituted.

Reply:
This is also used for all the calls to iree_compile and iree_run_module where we specify the directory to run the command from